src/assessment.js

```diff
@@ -49,5 +49,8 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  console.log(this);
 };
+
+deathstar.fireOnAlderaan(alderaan);
```

Comment:
Sorry, this is not correct. Do some Google searching and see fi you can come up with the answer.